src/components/Tool/NavToolList/Item/Item.tsx

```diff
@@ -22,29 +25,46 @@ interface Props {
   url: string;
   title: string;
   isSelected: boolean;
+  loading: boolean;
+  needsAttention: boolean;
+  totalCount: number;
+  toolId: string;
 }
 
-export const Item = ({ url, title, isSelected }: Props): ReactElement => {
-  const accountListId = useAccountListId();
+export const Item = ({
+  url,
+  title,
+  isSelected,
+  loading,
+  needsAttention,
+  totalCount,
+  toolId,
+}: Props): ReactElement => {
   const { classes } = useStyles();
+  const accountListId = useAccountListId();
+  const { t } = useTranslation();
 
   return (
     <NextLink
       href={`/accountLists/${accountListId}/tools/${url}`}
       scroll={false}
     >
-      <ListItem
-        button
-        selected={isSelected}
-        className={isSelected ? classes.liSelected : classes.liButton}
-      >
+      <ListItem button selected={isSelected}>
         <ListItemText
+          data-testid={`${toolId}-list-item`}
           primaryTypographyProps={{
             variant: 'subtitle1',
             color: 'textPrimary',
           }}
-          primary={title}
+          primary={t(title)}
```

Comment:
Instead of localizing the title here, let's localize it where it is defined, which I believe is `src/components/Tool/Home/ToolList.ts`. With a few rare exceptions, you always want a static string as the argument to `t()`, not an expression or a variable name. That way `yarn extract` can pick up the translation string and make sure it gets translated in OneSky.

Reply:
You can `import i18n from 'src/lib/i18n';` and then use `i18n.t` instead of the `t` from `useTranslation`.